examples/openvino/stable-diffusion/train_text_to_image_qat.py

```diff
@@ -1,1092 +0,0 @@
-#!/usr/bin/env python
-# coding=utf-8
-# Copyright 2023 The HuggingFace Inc. team. All rights reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-
-import argparse
-import itertools
-import logging
-import math
-import os
-import random
-from copy import deepcopy
-from functools import partial
-from io import BytesIO
-from pathlib import Path
-from typing import Iterable, Optional
-
-import numpy as np
-import requests
-import tomesd
-import torch
-import torch.nn.functional as F
-import torch.utils.checkpoint
-from accelerate import Accelerator
-from accelerate.logging import get_logger
-from accelerate.utils import ProjectConfiguration, set_seed
-from datasets import load_dataset
-from diffusers import DDIMScheduler, DDPMScheduler, DiffusionPipeline, LMSDiscreteScheduler, StableDiffusionPipeline
-from diffusers.optimization import get_scheduler
-from huggingface_hub import HfFolder, Repository, whoami
-from nncf import NNCFConfig
-from nncf.common.logging import nncf_logger
-from nncf.torch import create_compressed_model, register_default_init_args
-from nncf.torch.initialization import PTInitializingDataLoader
-from nncf.torch.layer_utils import CompressionParameter
-from PIL import Image
-from requests.packages.urllib3.exceptions import InsecureRequestWarning
-from torchvision import transforms
-from tqdm import tqdm
-
-from optimum.exporters.openvino import export_from_model
-
```

Comment:
why we remove the QAT example ? is QAT for SD models support  deprecated ?

Reply:
I think we can safely remove it because the sample and the workflow itself are outdated. We can get better and unbiased results using post-training methods only with no fine-tuning and GPUs required.